src/helpers/helpers.dom.ts

```diff
@@ -53,7 +53,12 @@ function parseMaxStyle(styleValue: string | number, node: HTMLElement, parentPro
 const getComputedStyle = (element: HTMLElement): CSSStyleDeclaration =>
   element.ownerDocument.defaultView.getComputedStyle(element, null);
 
-export function getStyle(el: HTMLElement, property: string): string {
+// @ts-ignore
```

Comment:
In my opinion the ignore should not be used, instead the error should be fixed

Reply:
Completed in e50c4c5